OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -76,8 +76,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
         pager.setOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+
+            private Boolean  firstTime = true;
```

Comment:
There's an extra space here that's not needed.

Reply:
check